pages/signup-page/signup-page.css

```diff
@@ -8,3 +8,38 @@
 .navbar{
     grid-row: 1/2;
 }
+.signup-container{
+    grid-row: 2/3;
+    height: 85vh;
+}
+.signup-heading{
+    text-align: center;
+}
+.form-wrapper{
+    margin: 2rem;
+    padding: 2rem;
+    display: flex;
+    flex-direction: column;
+    max-width: 23rem;
+    margin-left: auto;
+    margin-right: auto;
+    box-shadow: 0px 0px 10px 4px rgb(0 0 0 / 5%);
+}
+.checkbox-div{
+    padding: 0.5rem;
+    display: flex;
+    justify-content: flex-start;
+    gap: 0.5rem;
+}
+.input{
+    padding: 0.5rem;
+    caret-color: var(--color-primary);
+    border: 2px solid var(--color-primary);
+    font-size: 1rem;
+    width: 100%;
+}
+.btn-signup{
+    width: 100%;
+    margin: 1rem 0rem 0rem 0rem;
+    border-radius: 0rem;
```

Comment:
use proper tag like margin-top & remove unnecessary code line 44

Reply:
Line 44 was added to remove the side one side effect from other CSS, will take care in future.